Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.yml

```diff
@@ -68,17 +68,21 @@ script:
     - defaultValue: '50'
       description: Limit of results to retrieve.
       name: limit
-    - defaultValue: '1'
-      description: Page number.
+    - description: Page number.
       name: page
+      hidden: true
+    - description: Amount of results per request. Value can be range between 1 and 500. When only page_size is given the first page results will be fetched.
```

Comment:
Amount of results per request. Value can be between 1 and 500. When only page_size is given, the first page results will be fetched.